src/components/form.jsx

```diff
@@ -26,23 +26,23 @@ const Form = () => {
   };
   return (
     <div className="form p-4">
-      <h3 className="book-title font-bold text-2xl mb-4">Add new book</h3>
+      <h3 className="book-title font-bold text-2xl mb-4 text-gray-400">Add new book</h3>
       <form onSubmit={submit}>
         <input
-          className="border border-gray-400"
+          className="border border-gray-400 w-1/2 mr-8"
```

Comment:
- [ ]  Kindly fix the alignment issue on the form section 😄 , as represented in the template design 👍 

### Your design
![Capture2](https://github.com/mhnazary/bookstore/assets/80458199/55d77262-fd78-4b29-9d4b-27787690fc31)

### Template design
![Capture22](https://github.com/mhnazary/bookstore/assets/80458199/b2407661-b7e5-4a25-bb82-fdfd8d7e73b7)